x/cdp/client/rest/query.go

```diff
@@ -16,10 +18,12 @@ import (
 func registerQueryRoutes(cliCtx context.CLIContext, r *mux.Router) {
 	r.HandleFunc("/cdp/accounts", getAccountsHandlerFn(cliCtx)).Methods("GET")
 	r.HandleFunc("/cdp/parameters", getParamsHandlerFn(cliCtx)).Methods("GET")
+	r.HandleFunc("/cdp/savingsratedist", getSavingsRateDistributedHandler(cliCtx)).Methods("GET")
```

Comment:
the naming should be consistent with the rest of the endpoints (camelCase)

Reply:
https://github.com/Kava-Labs/kava/pull/644/commits/3a59e79d6cf41cda60c1f5a6819f028d30b6f47a